cookiecutter-django-app/hooks/post_gen_project.py

```diff
@@ -55,20 +51,12 @@ def move(src, dest):
 
 extra_context["placeholder_repo_name"] = python_placeholder_repo_name
 
-if import_template_from_github:
-    directory = "python-template"
-    cookiecutter(
-        'git@github.com:edx/edx-cookiecutters.git',
-        extra_context=extra_context,
-        no_input=True,
-        directory=directory
-        )
-else:
-    cookiecutter(
-        os.path.join(EDX_COOKIECUTTER_ROOTDIR, 'python-template'),
```

Comment:
Instead of joining the sub-template's name with the base directory, we can just use the base path and pass the `directory=` kwarg, like with github.